arangod/Aql/OptimizerRulesFeature.cpp

```diff
@@ -389,6 +389,11 @@ void OptimizerRulesFeature::addRules() {
                OptimizerRule::removeSatelliteJoinsRule,
                OptimizerRule::makeFlags(OptimizerRule::Flags::CanBeDisabled,
                                         OptimizerRule::Flags::ClusterOnly));
+
+  registerRule("remove-distribute-nodes", removeDistributeNodesRule,
```

Comment:
Do we have a list of rules in docs anywhere?

Reply:
https://github.com/arangodb/docs/blob/master/3.7/aql/execution-and-performance-optimizer.md (and there needs to be a PR to docs that documents this rule)